vsintegration/src/FSharp.Editor/CodeFix/AddOpenCodeFixProvider.fs

```diff
@@ -108,6 +108,7 @@ type internal FSharpAddOpenCodeFixProvider
         assemblyContentProvider: AssemblyContentProvider
     ) =
     inherit CodeFixProvider()
+    let fixableDiagnosticIds = ["FS0039"]
```

Comment:
I didn't say that ;-) 
Just saying if we find a common pattern than this is beneficial.